airflow/providers/google/CHANGELOG.rst

```diff
@@ -148,6 +148,66 @@ Breaking changes
 
 * ``BigQueryHook.run_grant_dataset_view_access`` Remove ``source_project``. Please use ``project_id``
 
+Details of breaking changes:
```

Comment:
Is this section just to add PR numbers?
Its not so nice to read.
I can raise PR to add the PR numbers to all entries I wrote

Reply:
It helps with automation (but I am waiting for some responses till tomorrow and I can move the numbers too.